src/Intl/UnitedNationsBundle.php

```diff
@@ -7,15 +7,17 @@
 
 class UnitedNationsBundle
 {
+    private static $countries;
+
     /**
      * Returns the list of the United Nations member states translated in a given locale.
-     *
-     * @param string $locale
-     *
-     * @return array
      */
-    public static function getCountries($locale = 'fr'): array
+    public static function getCountries(?string $locale = 'fr'): array
     {
+        if (self::$countries) {
+            return self::$countries;
```

Comment:
This is not logical in the sense of this class. It does not cost much to use an array.